### DISCUSSION TITLE
What is the difference between this package and EconML?

### DISCUSSION THREAD

**Discussion by adam2392** [uber/causalml#685] [Q&A]
Hi,

I'm perusing GitHub and I saw this package. Based on perusing both EconML and causalML's documentation, I am unclear what are the differences. 

Are there any? If so, is there a reason there are two packages? I'm just trying to understand when to use what.

I saw this [post](https://github.com/uber/causalml/discussions/413), but it didn't really provide me any information on this.

**Comment by jeongyoonlee** [comment]
EconML and CausalML provide various conditional average treatment effect (CATE) estimation methods along with evaluation and visualization methods for those estimators. Both packages were released as open-source in 2019.

In terms of differences,
- Each provides estimators that the other doesn't. e.g., EconML supports DML/DR OrthoForest and Deep/Ortho/DML/ITT IV, while CausalML supports Uplift Forest, CEVAE, DragonNet and TMLE.
- EconML is part of a bigger Py-Why echo system and defers tools for non-CATE estimation causal inferences to other packages such as dowhy and causal-learn.
- CausalML is a standalone tool designed to support the end-to-end process, including sensitivity analysis and policy optimization. It also provides methods for the traditional propensity score matching as it's been used widely in the industry.

The EconML and CausalML teams work together to evangelize causal machine learning. We have organized the causal machine learning [tutorial](https://causal-machine-learning.github.io/kdd2021-tutorial/) and [workshop](https://causal-machine-learning.github.io/kdd2023-workshop) at KDD in the past, and plan to continue working together.

Hope this helps.
